src/main/java/com/notably/logic/parser/DeleteCommandParser.java

```diff
@@ -37,7 +41,7 @@ public DeleteCommandParser(Model notablyModel, CorrectionEngine<AbsolutePath> pa
                 || !argMultimap.getPreamble().isEmpty()) {
             title = args.trim();
             if (title.isEmpty()) {
-                throw new ParseException("Path cannot be empty");
+                throw new ParseException(ERROR_EMPTY_PATH);
             }
```

Comment:
Actually, i'd say we move this `title.isEmpty()` check after the `if-else` block. This is because, even if you extract the title of the `argMultimap`, it can be empty!

Reply:
Same thing for other `CommandParser`s!

Reply:
Done!